client/client-multi/client-objective-c/Kaa/persistance/KaaClientPropertiesState.m

```diff
@@ -52,12 +53,13 @@ @interface KaaClientPropertiesState ()
 @property (nonatomic, strong) KeyPair *keyPair;
 @property (nonatomic) BOOL isConfigVersionUpdated;
 @property (nonatomic) BOOL hasUpdate;
+@property (nonatomic) BOOL isAutogeneratedKeys;
```

Comment:
Because of grammar issue I'd suggest to rename this parameter everywhere to smth like isKeyPairAutogenerated, generateKeyPair etc.